### DISCUSSION TITLE
[Bug]: When using 'gemini-pro' and 'gemini-pro-vision', if you ask questions in Chinese, the Chinese responses you receive are likely to be garbled

### DISCUSSION THREAD

**Discussion by xiaosaaaa** [danny-avila/LibreChat#1398] [Troubleshooting]
### What happened?

When using 'gemini-pro' and 'gemini-pro-vision', if you ask questions in Chinese, the Chinese responses you receive are likely to be garbled，Other models such as chatgpt and bing are normal.

### Steps to Reproduce

1. ask questions in Chinese
2. Get a reply
3. Garbled code generation

### What browsers are you seeing the problem on?

Chrome

### Screenshots

![dd](https://github.com/danny-avila/LibreChat/assets/115446601/dc398a5e-dd6c-46c1-a4d3-8f628e10e534)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by pzcn** [comment]
same
and these don’t seem to be sensitive words

**Comment by xiaosaaaa** [comment]
> same and these don’t seem to be sensitive words

yes,

> be sensitive words

yes, I've turned off the censor function for sensitive words.

**Comment by danny-avila** [comment]
This has nothing to do with the app as your text is not tampered with. This should be reported to google And probably due to some Unicode handling

**Comment by danny-avila** [comment]
Related google issue: https://github.com/google/generative-ai-js/issues/35
